app-nordigen/typeDefs.js

```diff
@@ -0,0 +1,176 @@
+/**
```

Comment:
Can you migrate this to a `.ts` file? I think it’ll be easier to read that way, and then you can do `import('./type-defs').Foo` in a JSDoc comment.

Reply:
Ok